scripts/automation/wait_and_collect_coderabbit.py

```
@@ -0,0 +1,31 @@
+
+out = subprocess.check_output(
+    ["gh", "pr", "list", "--state", "open", "--head", branch, "--json", "number"],
+    text=True,
+)
+pr = json.loads(out)[0]["number"]
+
+wait_result = subprocess.run(
+    ["scripts/automation/wait_coderabbit.sh", str(pr), str(wait_mins), status_file],
+    check=False,
+)
+subprocess.check_call([
```

Ruff (0.15.5)

[error] `subprocess` call: check for execution of untrusted input

(S603)